bin/verilator

```diff
@@ -2796,7 +2796,8 @@ are the same thread (i.e. the user's top C++ testbench runs on a single
 thread), but this is not required.
 
 The --trace-threads options can be used to produce trace dumps using multiple
-threads. This is orthogonal to --threads.
+threads. If --trace-threads is set without --threads, then --trace-threads will
+imply --threads 1, i.e.: the support libraries will be thread safe.
```

Comment:
@wsnyder I added this as VerilatedMutex and friends need VL_THREADED. I have not noticed a performance hit with the library being thread safe and otherwise it's ifdef hell to only enable bits that VL_TRACE_THREADED would actually need. Unless you have an issue with this I will merge this when the test pass.

Reply:
I think this is fine.